community-membership.md

```diff
@@ -219,3 +219,33 @@ The following apply to the subproject for which one would be an owner.
 - Ensure a healthy process for discussion and decision making is in place.
 - Work with other maintainers to maintain the project's overall health and
   success holistically.
+
+### Resolving technical conflicts within a SIG
+
+From time to time, the Maintainers for a given OpenTelemetry SIG may be unable
+to reach consensus on a technical issue. While it's healthy and appropriate to
+make a sincere attempt to understand all points of view and consider the
+tradeoffs, it's also healthy to occasionally "disagree and commit."
+
+Within OpenTelemetry, SIG Maintainers are chosen specifically _because_ they
+are domain experts, so we would like to keep as much of the decision-making
+authority with the SIG Maintainers rather than immediately "escalating" to the
+overall OpenTelemetry-wide Technical Committee. As such, this is
+OpenTelemetry's process for resolving technical issues where Maintainers cannot
+reach consensus:
+
+1. The SIG Maintainers should succintly document the options under
+consideration as a GitHub issue within the SIG's respective repo (note that it
+is *not* required to document the complete framing and pros/cons – just the
+actual go-forward options themselves).
+2. Each SIG Maintainer must formally vote for their choice by commenting on
+that issue.
+3. The option receiving the most votes wins.
```

Comment:
If the issue can be solved by voting does it need to follow a formal process like this? The voting can happen in the SIG and I am not sure how making the voting formal helps unless for some reason we think we need to have a voting record.
I am not opposed to voting but would like to understand how we think formalizing it and recording as an issue helps.

Reply:
Because voting *is* a formal process where someone is bound to lose, so it is better to have it documented. If the SIG was able to reach the consensus without it they wouldn't need to reach for this mechanism.